js/index.js

```diff
@@ -1,7 +1,114 @@
+
+//this works if I remove the paragraphs, as I am counting white spaces for words. not sure how to catch the 2 words after a paragraph break.
+var wordCount = 0;
+var searchTerm = ` et `;
+var etCount = 0;
+var nextIndexOfTerm = lorem.indexOf(searchTerm); //first occurence of search term. returns -1 if none found.
+
+for (var i = 0; i < lorem.length; i++) {
+  if (lorem[i] === ` `) {
+    wordCount++;
+  }
+}
+wordCount++; //this is for the last word of the string which is not succeeded by a space.
+console.log(`Number of words in string: ${wordCount}`);
+
+//for the length of string, and if indexOf finds search term, count++ and update nextindex for search.
+for (var i = 0; i < lorem.length && nextIndexOfTerm !== -1; i++) {
+  etCount++;
+  nextIndexOfTerm = lorem.indexOf(searchTerm, nextIndexOfTerm + 1);
+}
+console.log(`Number of occurences of "${searchTerm}": ${etCount}`);
```

Comment:
Works fine :)
Another solution a bit more straight-forward would be to convert your string to an array and iterate over to see if any element of the array is equal to "et"